std-bits/base/src/main/java/org/enso/base/statistics/Variance.java

```diff
@@ -0,0 +1,25 @@
+package org.enso.base.statistics;
+
+public class Variance implements Statistic {
+  private final boolean population;
+
+  public Variance(boolean population) {
+    this.population = population;
+  }
+
+  public boolean isPopulation() {
+    return population;
+  }
+
+  @Override
+  public int order() {
+    return 2;
+  }
+
+  @Override
+  public double evaluate(long n, double[] sums) {
+    return n < (population ? 1 : 2)
+        ? Double.NaN
+        : (sums[1] - sums[0] * sums[0] / n) / (population ? n : n - 1);
```

Comment:
It is very terse.
Won't just creating separate branches for the two variants be much clearer to read?

```suggestion
    if (population) {
        return n < 1 ? Double.NaN : (sums[1] - sums[0] * sums[0] / n) / n;
    } else {
        return n < 2 ? Double.NaN : (sums[1] - sums[0] * sums[0] / n) / (n - 1);
    }
```